src/components/Discussion/Composer/docs.imports.js

```diff
@@ -66,6 +66,12 @@ export const CommentComposerPlayground = () => {
     actions: {
       openDiscussionPreferences: () => Promise.resolve({ ok: true })
     },
+    getLabel: text => {
```

Comment:
First impressions can be tough, but it felt like `getLabel` needs a home, like `actions.formatHelper`, same as `openDiscussionPreferences`.

Reply:
also in actions rein?

Reply:
jo